src/libretro/config/definitions/network.hpp

```diff
@@ -71,6 +71,24 @@ namespace MelonDsDs::config::definitions {
         MelonDsDs::config::values::AUTO
     };
 #endif
+    constexpr retro_core_option_v2_definition LanMacAddressMode {
+        config::network::MAC_ADDRESS_MODE,
+        "Network MAC Address Mode",
+        "MAC Address Mode",
+        "Configures how the emulated console's MAC address is set. "
+        "Changing this option might make local multiplayer impossible or block access to save files "
+        "in games that use the MAC address to prevent tampering of save files (i.e. Pokémon).\n"
+        "No relation to the direct mode interface. Changes take effect at next restart.\n"
+        "See https://github.com/jessetg/melonds-ds/blob/main/LanMultiplayer.md for more information.",
```

Comment:
I'd prefer if a URL shortener was used here, as players will have to enter this address manually. Also I'd rather have that information on the libretro docs, but that space isn't set up yet; I'll create a page before merging this PR.

Reply:
If a libretro docs page was set up, it would have URL https://docs.libretro.com/library/melondsds/ , which I think is short enough to not need a link shortener, but it's your choice